docs/gallery/cnot.ipynb

```diff
@@ -1384,15 +1384,83 @@
     "    main()\n",
     "```\n",
     "\n",
-    "</details>\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "\"hide-input\""
```

Comment:
To do - this only works locally. Need to find a way to hide the code block in `nbsphinx` while also executing it. 

![image](https://github.com/user-attachments/assets/647794f3-9899-4227-aa0c-85e5d01e017c)

The code block above does not execute the code